plugins/versions.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+const checker = require('license-checker');
+const process = require('process');
+const VError = require('verror');
+
+/**
+ * Hapi interface for plugin to return package list
+ * @method register
+ * @param  {Hapi.Server}    server
+ * @param  {Object}         options
+ * @param  {Function} next
+ */
+exports.register = (server, options, next) => {
+    const SD_REGEX = /^screwdriver-/;
```

Comment:
A comment example of what this regex is intended to match with would help with others debugging this in the future